src/components/IdsActionPanel.vue

```diff
@@ -69,6 +70,11 @@ const actionPanelStyle = reactive({
 .elevated {
   @include commonMixin;
   box-shadow:
-    var(--ids-smc-reference-container-effects-tw-shadow-horizontal-none) var(--ids-smc-reference-container-effects-tw-shadow-vertical-xxl) var(--ids-smc-reference-container-effects-tw-shadow-blur-xxxl) var(--ids-smc-reference-container-effects-tw-shadow-spread-xxs) var(--ids-smc-reference-container-effects-tw-shadow-color-dark-darker);
+    var(--ids-smc-reference-container-effects-shadow-horizontal-none) var(--ids-smc-reference-container-effects-shadow-vertical-xxl) var(--ids-smc-reference-container-effects-shadow-blur-xxxl) var(--ids-smc-reference-container-effects-shadow-spread-xxs) var(--ids-smc-reference-container-effects-shadow-color-dark-darker);
 }
-</style>
\ No newline at end of file
+
+//box-shadow: var(--container-effects-shadow-horizontal-none, 0px)
+//var(--container-effects-shadow-vertical-xxl, 25px)
+//var(--container-effects-shadow-blur-xxxl, 50px) 
+//var(--container-effects-shadow-spread-xxs, -12px) 
+//var(--container-effects-shadow-color-dark-default, rgba(0, 0, 0, 0.25));</style>
```

Comment:
The closing </style> element is commented out

Reply:
fixed